java/src/main/java/scenarios/crewcreation/CharacterCreation.java

```diff
@@ -119,102 +120,136 @@ private static void printMeanScoreFlavorText (int meanScore) {
-            }
-        } else if (abilityScoreName.equals(PILOTING)) {
-            if (abilityScoreValue == 18) {
-                System.out.println("Wowee! You could fit a strike ship through a garbage chute!");
-            } else if (abilityScoreValue >= 14) {
-                System.out.println("We should go for a race sometime, you've got some fancy tricks.");
-            } else if (abilityScoreValue >= 10) {
-                System.out.println("You could handle low-orbit well. Try it out sometime.");
-            } else if (abilityScoreValue == 9) {
-                System.out.println("Fairly average, but at least you can pilot a ship.");
-            } else if (abilityScoreValue >= 7) {
-                System.out.println("I've seen worse.");
-            } else if (abilityScoreValue >= 3) {
-                System.out.println("You had your hyperdrive license suspended for how long?!");
-            }
-        } else if (abilityScoreName.equals(REPAIR)) {
-            if (abilityScoreValue == 18) {
-                System.out.println("Golly! You could fix things before they even break!");
-            } else if (abilityScoreValue >= 14) {
-                System.out.println("Impressive... Now listen, I've got this sound coming from my kinetic barriers..");
-            } else if (abilityScoreValue >= 10) {
-                System.out.println("Spent a summer working in the engineering bay of a Cruiser, eh?");
-            } else if (abilityScoreValue == 9) {
-                System.out.println("At least I can trust you not to mess things up.");
-            } else if (abilityScoreValue >= 7) {
-                System.out.println("Guess you skipped that module at the Academy.");
-            } else if (abilityScoreValue >= 3) {
-                System.out.println("Give that wrench back right now.");
-            }
-        } else if (abilityScoreName.equals(CHARISMA)) {
-            if (abilityScoreValue == 18) {
-                System.out.println("Wowzers! You could charm the pants off of me!");
-            } else if (abilityScoreValue >= 14) {
-                System.out.println("You must have no trouble getting into those selective Nebular clubs.");
-            } else if (abilityScoreValue >= 10) {
-                System.out.println("Had more than a few ladies hoping you'd take 'em for a night-time cruise, eh?");
-            } else if (abilityScoreValue == 9) {
-                System.out.println("Average, but it's better than some can say.");
-            } else if (abilityScoreValue >= 7) {
-                System.out.println("You're not convincing anyone.");
-            } else if (abilityScoreValue >= 3) {
-                System.out.println(
-                        "I probably wouldn't even notice you exist, if I weren't being payed to talk to you..."
-                );
-            }
+        switch (abilityName) {
```

Comment:
You should use the constants I declared at the top of the class instead of these [String Literals](https://en.wikipedia.org/wiki/String_literal) for the cases.